Android.bp

```diff
@@ -22,8 +42,16 @@ mfx_cc_defaults {
 
         "-DANDROID",
         "-DMFX_C2_USE_PRIME",
+        "-DLIBVA_SUPPORT",
+        "-DLIBVA_ANDROID_SUPPORT",
```

Comment:
Can we add -DMFX_VERSION=2008 here? We need this variable to check if we can use onevpl api. http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/vendor/intel/mediasdk_c2/c2_utils/include/mfx_defs.h?r=16901798#30

Reply:
oh, I forgot this one...add it now